pages/linux/pwgen.md

```diff
@@ -0,0 +1,7 @@
+# pwgen
+
+> Generate passwords.
```

Comment:
According to their webpage, they are a tool to `generate pronounceable passwords`